src/base/map_info.ixx

```diff
@@ -425,6 +464,10 @@ export class MapInfo {
 		writer.write(supported_modes);
 		writer.write(game_data_version);
 
+		writer.write(default_cam_distance);
+		writer.write(max_cam_distance);
+		writer.write(min_cam_distance);
```

Comment:
default & max should be written for >= 32

min should be written for >= 33

Reply:
We always write latest, note write_version being 33